### PR TITLE
feat: remove_video_Honor

### PR DESCRIPTION
**Pull request by huajijam**
移除视频内的荣誉榜 (入站必刷、每周必看、全站排行榜、4K频道精选等)

### CODE REVIEW

File: app/src/main/java/me/iacn/biliroaming/hook/ProtoBufHook.kt [hunk 1]
```diff
@@ -143,7 +147,7 @@ class ProtoBufHook(classLoader: ClassLoader) : BaseHook(classLoader) {
                         callMethod("clearListenForegroundImage")
                         callMethod("clearStoryBackgroundImage")
                         callMethod("clearStoryForegroundImage")
-                        val tabUrl = "bilibili://root?tab_name=我的"
+                        val tabUrl = "bilibili://root?tab_name=�ҵ�"
```
[the remark this answers is not in the document]
Comment (huajijam) [reply app/src/main/java/me/iacn/biliroaming/hook/ProtoBufHook.kt]: cause by patches bcz my local branch have conflic,fixed

**Review by yujincheng08 — APPROVED** [review]

**Review by yujincheng08 — APPROVED** [review]
好耶

File: app/src/main/java/me/iacn/biliroaming/hook/ProtoBufHook.kt [hunk 2]
```diff
@@ -8,6 +8,7 @@ class ProtoBufHook(classLoader: ClassLoader) : BaseHook(classLoader) {
         val purifyCity = sPrefs.getBoolean("purify_city", false)
         val removeRelatePromote = sPrefs.getBoolean("remove_video_relate_promote", false)
         val removeRelateOnlyAv = sPrefs.getBoolean("remove_video_relate_only_av", false)
+        val removeHonor = sPrefs.getBoolean("remove_video_Honor", false)
```
Comment (yujincheng08) [inline app/src/main/java/me/iacn/biliroaming/hook/ProtoBufHook.kt:11]: H 小写

**Review by yujincheng08 — DISMISSED** [review]
好耶

File: app/src/main/java/me/iacn/biliroaming/hook/ProtoBufHook.kt (hunk 2 above)
Comment (huajijam) [reply app/src/main/java/me/iacn/biliroaming/hook/ProtoBufHook.kt]: fixed

File: app/src/main/res/xml/prefs_setting.xml [hunk 3]
```diff
@@ -396,6 +396,11 @@
             android:key="remove_video_UgcSeason"
             android:summary="@string/remove_video_UgcSeason_summary"
             android:title="@string/remove_video_UgcSeason_title" />
+        <SwitchPreference
+            android:dependency="hidden"
+            android:key="remove_video_honor"
+            android:summary="@string/remove_video_Honor_summary"
```
Comment (yujincheng08) [inline app/src/main/res/xml/prefs_setting.xml:402]: H 小写

**Review by yujincheng08 — APPROVED** [review]
好耶

File: app/src/main/res/xml/prefs_setting.xml (hunk 3 above)
Comment (huajijam) [reply app/src/main/res/xml/prefs_setting.xml]: now? pup

**Pull request merged**